creator-node/src/routes/tracks.js

```diff
@@ -313,8 +313,7 @@ module.exports = function (app) {
       const existingTrackEntry = await models.Track.findOne({
         where: {
           cnodeUserUUID,
-          blockchainId: blockchainTrackId,
-          coverArtFileUUID
+          blockchainId: blockchainTrackId
```

Comment:
What was happening here is we were using the coverArtFileUUID for the existing track check which means if the cover art was changed, this check would fail and it would treat this as a new upload and fail because there were no unassociated files